### PR TITLE
kubenurse configuerable resources

### PR DESCRIPTION
**Pull request by myaser**
teapot/issues#1950

### CODE REVIEW

**Comment by myaser** [comment]
👍🏻

File: cluster/manifests/kubenurse/daemonset.yaml [hunk 1]
```diff
@@ -56,11 +56,11 @@ spec:
           image: "container-registry.zalando.net/teapot/kubenurse:v1.7.0-histogram-main-5"
           resources:
             requests:
-              cpu: "40m"
-              memory: "100Mi"
+              cpu: "{{.Cluster.ConfigItems.network_monitoring_cpu}}"
```
Comment (demonCoder95) [inline cluster/manifests/kubenurse/daemonset.yaml]: I think the name is `network_monitoring_daemonset_cpu` same for memory?

Comment (myaser) [reply cluster/manifests/kubenurse/daemonset.yaml]: fixed

**Comment by demonCoder95** [comment]
👍

**Comment by myaser** [comment]
👍🏻

**Pull request merged**